internal/httpcli/client.go

```diff
@@ -467,6 +468,7 @@ func TracedTransportOpt(cli *http.Client) error {
 	}
 
 	cli.Transport = &policy.Transport{RoundTripper: cli.Transport}
+	cli.Transport = otelhttp.NewTransport(cli.Transport)
```

Comment:
The docs say we don't need it, but if we want to be extra sure:

```suggestion
	cli.Transport = otelhttp.NewTransport(cli.Transport
	    otelhttp.WithPropagators(otel.GetTextMapPropagator()))
```

The propagator gets initialized to one that propagates spans in OpenTracing, Jaeger, and the standard W3C formats